content/help/ops/setup/webhook/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Dynamic Admission Webhooks Overview
+description: Provides a general overview of Istio's use of Kubernetes webhooks and the related issues that can arise.
+weight: 10
+---
+
+From [Kubernetes mutating and validating webhook mechanisms](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/):
+
+> Admission webhooks are HTTP callbacks that receive admission requests
+and do something with them. You can define two types of admission
+webhooks, validating admission Webhook and mutating admission
+webhook. With validating admission Webhooks, you may reject requests
+to enforce custom admission policies. With mutating admission
+Webhooks, you may change requests to enforce custom defaults.
+
+The webhook setup guides assuming general familiarity with Kubernetes
```

Comment:
What are the "webhook setup guides"? Should that be singular? And should this be a link to a doc page?

Reply:
Its a reference to the subsequent validation and injection webhook guides.